docs/report/report.tex

```diff
@@ -189,174 +189,164 @@ \subsection{Deployment Cycle}
+Um zukünftige Probleme gar nicht erst entstehen zu lassen, hatte sich das Team vorgenommen, entstandene Probleme nicht erst im Standup, sondern früher zu kommunizieren.
+Genauso sollte neu Gelerntes besser kommuniziert werden, um die Arbeit effizienter zu gestalten.
+
+\subsection{Meilenstein 4}
+Meilenstein vier lag innerhalb der Weihnachtszeit und wurde deswegen mit einem größeren Zeitraum vom 17.12.2020 bis 14.01.2021 geplant.
+Mit diesem Meilenstein begann die funktionale und visuelle Umsetzung der Hauptfunktionen.
+\subsubsection{Ziel}
+Wie auch beim vorherigen Meilenstein war wieder hier der Fokus auf den Kundenwunsch, dass die ersten Funktionalitäten umgesetzt werden.
+Dass umfasste das Anlegen von eigenen Tags, also dem Scannen von Tags, vergeben eines eigenen Namens, auswählen von Musik und einem eigenen Bild, sowie das bearbeiten bereits angelegter Tags.
+Auch sollten für den nächsten Schritt die Mockups erstellt werden, wie ein Nutzer den Lautsprecher zur Wiedergabe auswählt sowie seine eigene \textit{Leek Box} initial einrichtet.
+\subsubsection{Probleme}
+Zu diesem Meilenstein sind vermehrt größere Probleme aufgetreten.
+Die Planung und Einschätzung der umsetzbaren Tickets wurde schlecht eingeschätzt.
+Es wurde sich zu sehr in Details verloren, sodass der Fokus auf die wichtigen Tickets verloren ging.
+Das führte dazu, dass am Ende viele Tickets halbfertig und/oder nur schlecht getestet implementiert waren.
+\subsubsection{Lösungen}
+Kurzfristig wurde eine Team interne Evaluation der einzelnen Mitglieder durchgeführt.
+Das half einerseits dabei, dass Probleme gelöst und die Teammotivation sowie die Produktivität stieg, als auch das generelle Teamklima weiter gebessert wurde.
+\subsubsection{Product Increment}
+Das Produkt wurde um die Grundfunktionen erweitert.
+Es ist nun möglich, Tags anzulegen und entsprechend für diese Namen, Musik und ein Bild einzustellen.
+Genauso ist es dem Nutzer nun möglich, alle angelegten Tags zu durchsuchen.
+Es wurde weiterhin auch das Design Mockup von dem von Macio gestellten Designer in Vue größtenteils umgesetzt.
+Gleichzeitig wurden hierbei auch für eine leichtere Weiterarbeit jegliche Komponenten als möglichst allgemeine Vue Komponenten angelegt, um ein einheitliches Design zu ermöglichen.
+Für die Entwickler wurden auch die Pull Request Tests um Typescript Support erweitert.
+\subsubsection{Retrospektive}
+Besonders gut gefiel dem Team das Teamklima, vor allem des ehrliche Miteinander.
+Trotz Komplikationen und Probleme konnte am Ende dem Kunden ein schönes Product Increment geliefert werden.
+Pull Requests und Tickets wurden besser beschrieben, sodass Probleme besser verständlicher waren.
+Vor allem durch das interne Team Review wurde die Zusammenarbeit weiter gebessert.
+Das Team konnte weiterhin das Wissen mit Vue ausbauen, vor allem zu Reaktivität, Events und Kommunikation zwischen Komponenten.
```

Comment:
```suggestion
Das Team konnte das Wissen zu Vue, vor allem die Themen Reaktivität, Events und Kommunikation zwischen Komponenten ausbauen.
```

Reply:
umgesetzt in https://github.com/project-leek/project-leek/pull/409/commits/db7fedecbe3f12f759a2c504c4b9f0e5a7061fee